packages/galactica-types/src/merkleProof.ts

```diff
@@ -5,7 +5,7 @@ export type MerkleProof = {
   leaf: string;
   // hashes of the branches on the side of the path
   pathElements: string[];
-  // interpreted as binary number. If a bit is set, it means that the path is the right part of the parent node.
-  pathIndices: number;
+  // The leafIndex can also be interpreted as binary number. If a bit is set, it means that the path is the right part of the parent node.
```

Comment:
It's important to mention the order, that the rightmost bit corresponds to leaf.